tools/GENERATE-OPENAPI.sh

```diff
@@ -15,23 +15,19 @@
 # limitations under the License.
 #
 
-#
-# Compile .proto files into the files needed to build the registry server and
-# command-line tools.
-#
-
-# uncomment to update dependency
-go get -u github.com/googleapis/gnostic/apps/protoc-gen-openapi
+set -e
 
-ANNOTATIONS="third_party/api-common-protos"
+if [ ! -d "third_party/api-common-protos" ]
+then
+  git clone https://github.com/googleapis/api-common-protos third_party/api-common-protos
+fi
 
-PROTOS=( \
-	google/cloud/apigee/registry/v1/registry_models.proto \
-	google/cloud/apigee/registry/v1/registry_service.proto \
+SERVICE_PROTOS=(
```

Comment:
I agree with splitting these generation steps into separate scripts, but notice that the declarations (SERVICE_PROTOS, ALL_PROTOS) are repeated across multiple files. Since we seem to require that these scripts be run from the repo root, I think we could move these declarations into a common file ("tools/PROTOS.sh"?) and include that from the scripts - could you make this change to reduce duplication and the possibility of accidental inconsistency?